version.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	_ "embed"
+	"encoding/json"
+	"runtime/debug"
+)
+
+var version string
+
+//go:embed version.json
+var versionJSON []byte
+
+func init() {
+	// Read version from embedded JSON file.
+	var verMap map[string]string
```

Comment:
nit: this could be a struct
```suggestion
	var v struct {
		Version string `json:"version"`
	}
```

Reply:
done